tutorials/ros_ign_demo.md

```diff
@@ -0,0 +1,75 @@
+# ROS Integration
+
+In this tutorial we will learn how to Integrate ROS with Ignition. We will establish communication between them. This can help in many aspects, we can receive data or commands from ROS and apply it to ignition and vice versa.
+
+## ros_ign_bridge
+
+`ros_ign_bridge` provides a network bridge which enables the exchange of messages between ROS2 and Ignition Transport. Its support is limited to only certain message types. Please, check this [README](https://github.com/ignitionrobotics/ros_ign/blob/ros2/ros_ign_bridge/README.md) to verify if your message type is supported by the bridge.
+
+## Requirements
+
+To follow up with this tutorial make sure to have the following installed:
+
+* [ROS2 Foxy](https://index.ros.org/doc/ros2/Installation/Foxy/)
+* [Ignition Citadel](https://ignitionrobotics.org/docs/citadel)
+* [ros_ign_bridge](https://github.com/ignitionrobotics/ros_ign/blob/ros2/ros_ign_bridge/README.md#building-the-bridge-from-source)
+
+## Bidirectional communication
+
+We can initialize a bidirectional bridge so we can have ROS as the publisher and Ignition as the subscriber or vice versa.
+
+For example:
+
+```
+ros2 run ros_ign_bridge parameter_bridge /TOPIC@ROS_MSG@IGN_MSG
+```
+
+The command is simple `ros2 run ros_ign_bridge parameter_bridge` to run the `parameter_bridge`. Then we specify our topic `/TOPIC` over which the messages will be sent. The first `@` symbol delimits the topic name from the message types. Following the first `@` symbol is the ROS message type.
+
+The ROS message type is followed by an `@`, `[`, or `]` symbol where:
+
+* `@`  for a bidirectional bridge.
+* `[`  for a bridge from Ignition to ROS.
+* `]`  for a bridge from ROS to Ignition.
+
```

Comment:
How about finishing this section with a link to these other examples:

https://github.com/ignitionrobotics/ros_ign/blob/ros2/ros_ign_bridge/README.md#example-1a-ignition-transport-talker-and-ros-2-listener